book/intro.md

```diff
@@ -1,6 +1,28 @@
 (intro)=
+# A tutorial for 'Advancing ecosystem carbon flux research'
+
+This book helps to explore tools and datasets, that are useful for researchers working topics ranging from processes in the carbon cycle to machine learning tools and remote sensing. This book will be tested within the ["Advancing ecosystem carbon flux research"](https://www.lorentzcenter.nl/advancing-ecosystem-carbon-flux-research.html) at the Lorentz Center between 14 - 18 October 2024 and be used to inform the discussion on research priorities for the terrestrial carbon cycle and technical developments for research in the next decade. 
+
+The aim is for the concepts covered during the workshop to open the discussion on openess and reproducibility with the community and to introduce researchers to some useful tools and concepts that can be transfered to their research. 
+
 # Working with modern (cloud) data formats
+Geospatial data are becoming increasingly large and complex and as a result, downloading data has become increasingly impractical. Cloud-optimized geospatial formats present an alternative to downloading data, that is more efficient, flexible and scalable. Additionally, it allows the user to more easily download subsets of data, rather than the entire dataset.
+
+An overwiew of the common formats and the advantages of cloud-optimised formats can be found [here](https://guide.cloudnativegeo.org/)
+
+## Zarr
+Currently, many datasets, such as FLUXNET are stored in CSV or netcdf/hdf5 formats, which need to be downloaded by the user from a database requiring an account. This process can be quite time consuming and make it difficult to replicate research if the version, variables used etc are not adequately documented. 
```

Comment:
Maybe add some info here on how working with Zarr (as an end user) is quite similar to netCDF

Here's a nice overview of differences: https://help.marine.copernicus.eu/en/articles/8176692-how-to-choose-between-netcdf-and-zarr-format-using-the-toolbox#h_04479906c1

Reply:
I maybe addressed this although perhaps it could be more clear.